adoc/chapters/programming_interface.adoc

```diff
@@ -4917,6 +4917,8 @@ info::event_command_status::complete
 ----
    a@ Indicates that the command has finished running on the device.
       Attempting to wait on such an event will not block.
+      
+      Synchronization: Equivalent to event::wait().
```

Comment:
```suggestion
      Synchronization: Equivalent to [code]#event::wait()#.
```

The wording looks fine to me, but I'm on the fence about whether it should be specified here, or whether we need to make it clear somehow that it is the call to `get_info` which has the synchronization behavior, and only if the return value is `complete`?

Perhaps the easiest thing to do is to make the change as you suggested, and then rewrite this section later (removing the table, etc).

Reply:
Agree.  The "no-table" allows for more flexibility.
At least with the PR it's a little better than before (we try to explain), and in a new PR we can make it crystal clear :')

We can see what other people think in the following WG call 🤞🏽